package.json

```diff
@@ -20,7 +20,7 @@
   },
   "homepage": "https://github.com/ksachdeva/angular2-swing#readme",
   "dependencies": {
-    "swing": "^4.3.0"
+    "swing": "^3.1.0"
```

Comment:
@mashhoodr @gajus I always assumed that running ```npm view swing version``` or whatever package name you're using in npm will always return the latest version. In this case running that command returns 3.1.0. Maybe the npm versions are different to the Git versions.